atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -59,7 +59,19 @@ private CassandraKeyValueServices() {
         // Utility class
     }
 
-    static void waitForSchemaVersions(Cassandra.Client client, String tableName, int schemaTimeoutMillis)
+    static void waitForSchemaVersions(
+            Cassandra.Client client,
+            String tableName,
+            int schemaTimeoutMillis)
+            throws TException {
+        waitForSchemaVersions(client, tableName, schemaTimeoutMillis, false);
+    }
+
+    static void waitForSchemaVersions(
+            Cassandra.Client client,
+            String tableName,
+            int schemaTimeoutMillis,
+            boolean allowUnresponsiveNode)
             throws TException {
         long start = System.currentTimeMillis();
         long sleepTime = INITIAL_SLEEP_TIME;
```

Comment:
The version size check below passing if versions map is empty strikes me as odd.  Not really related though.
```
             if (versions.size() <= 1) {
```